tactile_calibration/tactile_calibration_tools/src/tactile_calibration_tools/calibration_utils.py

```diff
@@ -5,7 +5,6 @@
 import matplotlib.pyplot as plt
 from datetime import datetime
 import os
-#pip install pwlf
```

Comment:
I add this comment again as part of the review. Could we not keep that ? it was not left there by mistake. It helps co-workers...

Reply:
This type of doc doesn't belong into code in my opinion. You have (correctly) mentioned that in the [README.md](https://github.com/ubi-agni/tactile_toolbox/pull/2/files#diff-c4eb798bc949c4afdb28f8da46a92d82R43).

Reply:
Well, we need to change CMakeLists.txt to install the readme.  Usage is part of the help of the 2 apps, so readme was not installed (and is rarely in our packages), but the functions will not even show the help if this lib is missing. What is the common procedure on Readme for packages in ROS ?

Reply:
The readme is shown as soon you open the github page. This should be the starting point for all users, shouldn't it?
I don't see a need to install the readme. Where would you want to install it?

Reply:
No idea where to install a readme, this is why I ask. The customers will follow specialized instructions provided via a product manual so should have the info already. It seemed convenient to have it in the code that is the only thing a user easily has access to when installed. I would not do this for a system available package (for which a rosdep exists), here was a particular problem for which the dependency is not clearly appearing in other places (like package.xml).
Can we make the import optional, and warn the user to install this if missing ?